openshift/openshift.go

```diff
@@ -36,7 +36,7 @@ var (
 
 func ExportAsTemplate(filter *ResourceFilter, exportOptions *cli.ExportOptions) (string, error) {
 	ret := ""
-	args := []string{"export", "--as-template=tailor", "--output=yaml"}
+	args := []string{"get", "-o", "yaml", "--export"}
```

Comment:
I'd prefer `args := []string{"get", "--output=yaml", "--export"}`, as I think it is easier to see that `output` and `yaml` are acutally one param.